Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -303,28 +298,28 @@ namespace uWS {
             return 0;
         }
 
-        static inline size_t formatMessage(char* dst, const char* src, size_t length, OpCode opCode,
+        static inline size_t formatMessage(std::byte* dst, const char* src, size_t length, OpCode opCode,
                                            size_t reportedLength, bool compressed) {
             size_t messageLength;
             size_t headerLength;
             if ( reportedLength < 126 ) {
                 headerLength = 2;
-                dst[1]       = (char)reportedLength;
-            } else if ( reportedLength <= UINT16_MAX ) {
+                dst[1]       = (std::byte)reportedLength;
+            } else if ( reportedLength <= std::numeric_limits<uint16_t>::max() ) {
                 headerLength          = 4;
-                dst[1]                = 126;
+                dst[1]                = (std::byte)126;
                 *((uint16_t*)&dst[2]) = htons((uint16_t)reportedLength);
             } else {
                 headerLength          = 10;
-                dst[1]                = 127;
+                dst[1]                = (std::byte)127;
                 *((uint64_t*)&dst[2]) = htobe64(reportedLength);
             }
 
             int flags = 0;
-            dst[0]    = char((flags & SND_NO_FIN ? 0 : 128) | (compressed ? SND_COMPRESSED : 0));
-            if ( !(flags & SND_CONTINUATION) ) { dst[0] |= opCode; }
+            dst[0]    = (std::byte)((flags & SND_NO_FIN ? 0 : 128) | (compressed ? SND_COMPRESSED : 0));
+            if ( !(flags & SND_CONTINUATION) ) { dst[0] |= (std::byte)opCode; }
 
-            char mask[4];
+            std::byte mask[4];
```

Comment:
This looks weird before, and still weird now....does anything actually set these values or are they just...garbage?